app/validators/api/v4/teleconsultation_payload_validator.rb

```diff
@@ -9,10 +9,25 @@ class Api::V4::TeleconsultationPayloadValidator < Api::V3::PayloadValidator
     :deleted_at,
     :created_at
   )
+  attr_writer :request_user_id
 
   validate :validate_schema
+  validate :authorized_teleconsult_record, if: -> { record.present? }
+
+  def authorized_teleconsult_record
+    unless request_user.can_teleconsult?
```

Comment:
Since we're using the forgiving `find_by` downstairs, we should assume that `request_user` can be nil. Can we do `request_user&.can_teleconsult?`